test/factories/pledge.rb

```diff
@@ -3,5 +3,8 @@
     pregnancy
     pledge_type 'Soft'
     amount 100
+    after :build do |pledge, eval|
```

Comment:
Two quick things about this: 

a) not quite sure how this applies to the rest of the PR. Was it making tests fail?
b) We've generally been doing this in the form of `created_by { FactoryGirl.create(:user) }` -- see the `Patient` factory, for example. If we need to keep it, what do you think about swapping it over to use the standard format?